config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml

```diff
@@ -6,11 +6,12 @@ presubmits:
         preset-kind-volume-mounts: "true"
         preset-cluster-api-provider-vsphere-e2e-config: "true"
         preset-cloud-provider-vsphere-e2e-config: "true"
-      always_run: false
-      optional: true
       decorate: true
+      decoration_config:
+        timeout: 1h
+      run_if_changed: 'images/capi/(Makefile|ansible\.cfg)|images/capi/ansible/.*|images/capi/scripts/ci-ova\.sh|images/capi/packer/(config|goss|ova)/.*'
```

Comment:
Should we include `hack/*` in the list for both azure & ova?

Reply:
ooh, good catch. I meant to do that. I know it makes this line a bit unwieldy, but I think it would be better to list the exact files we care about for each builder. There's a lot of stuff under `hack/` that don't need to kick of a specific CI build. And some of them are provider specific. I'll just use the provider's `make deps` target as a template.

Reply:
Okay, I added the `hack/ensure*` scripts that each provider uses.